### DISCUSSION TITLE
A suggestion regarding custom invoices and stylesheets when updating BTCPay Server to the latest version

### DISCUSSION THREAD

**Discussion by clarketowson** [btcpayserver/btcpayserver#5737] [Ideas / Feature Requests]
Hi Everyone,

I updated my BTCPay Server from version 1.3.5 to the latest this morning and all went well thanks to some great tips and advice from @dennisreimann.  

One thing I noticed though was in each of my stores settings the option under Checkout "use the classic checkout" was not set to use the classic checkout.  On my server I have quite a number of stores set up so I had to go into each individually and change the option to use the classic checkout.  It wasn't a major pain for me at this stage but it will become one as time goes on and I set up more people on the server.

I wanted to suggest for future upgrades that a simple check me made on update that checks to see if your BTCPay Server stores have "Link to a custom logo" and/or "Link to a custom CSS stylesheet" set.  The check could then set the Checkout setting to use the classic checkout by default.

The work that I'm doing with BTCPay Server in Thailand for example I have a lot of individuals set up on my server with a lot more being on boarded later this year and next year so a check like this is very important for me and anyone else that has a large number of stores.  

I have spent a lot of time and effort customizing the look and feel of the classic invoices so they include the logos of the businesses that employ the people I'm setting up on my server as well as my own business logo.  I'm sure other users would have done the same thing.

With Very Best Regards,

Clarke

Clarke Towson, BCMS (Bachelor of Computer & Mathematical Science)
CEO
INTJ Billing
 
m: +61 432 359 166
a: 7 Cullen Court Spotswood Victoria 3015 AUSTRALIA
w: https://intjbilling.com/
e: [clarketowson@intjbilling.com](mailto:clarketowson@intjbilling.com)

Bitcoin Lightning Network Node Name: CHILLYCALENDAR 
Node Public Key: 025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693 
Status Page: https://intjbilling.com/INTJBillingStatusPage.php 

amboss.space: https://amboss.space/node/025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693 
1ml.com: https://1ml.com/node/025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693
Lightning network+: https://lightningnetwork.plus/nodes/025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693

**Comment by dennisreimann** [comment]
> I wanted to suggest for future upgrades that a simple check me made on update that checks to see if your BTCPay Server stores have "Link to a custom logo" and/or "Link to a custom CSS stylesheet" set. The check could then set the Checkout setting to use the classic checkout by default.

Thanks for the feedback, thats also a good point on resolving it. I'll check if we can prevent this from happening with future updates.

> I have spent a lot of time and effort customizing the look and feel of the classic invoices so they include the logos of the businesses that employ the people I'm setting up on my server as well as my own business logo.

For new stores, please check out the new branding options we've introduced for stores and invoices with the Checkout v2. I think they are going to make your life easier customizing the branding aspects. You can find that in the Store Settings > General > Branding section. These option will be honored on all public pages of the store, not just the checkout view.
